app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/utils/CheckYourBankAccountDetailsHelper.scala

```diff
@@ -55,7 +55,7 @@ class CheckYourBankAccountDetailsHelper @Inject() (implicit langs: Langs, messag
             List(
               ActionItem(
                 //TODO: Change routing for this link to go to the 'Enter account type' page
```

Comment:
Is this comment - //TODO: Change routing for this link to go to the 'Enter account type' page still needed?

Reply:
Nope ...

Reply:
Now removed